CompPlots.ipynb

```diff
@@ -23,15 +14,137 @@
+       " \u001b[0m        gasCur \u001b[0m \u001b[0m  0.1279 \u001b[0m \u001b[0m  0.0902 \u001b[0m \u001b[0m   0.0057 \u001b[0m \u001b[0m  0.0332 \u001b[0m \u001b[0m  7.2285 \u001b[0m \u001b[0m  1.0841 \u001b[0m \u001b[0m\n",
+       " \u001b[0m   internalize \u001b[0m \u001b[0m  0.0473 \u001b[0m \u001b[0m  0.0105 \u001b[0m \u001b[0m   0.0007 \u001b[0m \u001b[0m  0.0031 \u001b[0m \u001b[0m 11.1634 \u001b[0m \u001b[0m  1.0478 \u001b[0m \u001b[0m\n",
+       " \u001b[0m          kDeg \u001b[0m \u001b[0m  0.0212 \u001b[0m \u001b[0m  0.0048 \u001b[0m \u001b[0m   0.0003 \u001b[0m \u001b[0m  0.0026 \u001b[0m \u001b[0m  4.6812 \u001b[0m \u001b[0m  1.1543 \u001b[0m \u001b[0m\n",
+       " \u001b[0m          kRec \u001b[0m \u001b[0m  0.1071 \u001b[0m \u001b[0m  0.0119 \u001b[0m \u001b[0m   0.0008 \u001b[0m \u001b[0m  0.0031 \u001b[0m \u001b[0m 18.7062 \u001b[0m \u001b[0m  0.9973 \u001b[0m \u001b[0m\n",
+       " \u001b[0m pYinternalize \u001b[0m \u001b[0m  1.0975 \u001b[0m \u001b[0m  0.1366 \u001b[0m \u001b[0m   0.0086 \u001b[0m \u001b[0m  0.0163 \u001b[0m \u001b[0m  9.8463 \u001b[0m \u001b[0m  1.0011 \u001b[0m \u001b[0m\n",
+       " \u001b[0m         scale \u001b[0m \u001b[0m  0.0212 \u001b[0m \u001b[0m  0.0066 \u001b[0m \u001b[0m   0.0004 \u001b[0m \u001b[0m  0.0034 \u001b[0m \u001b[0m  5.1126 \u001b[0m \u001b[0m  1.1014 \u001b[0m \u001b[0m\n",
+       " \u001b[0m     scaleSurf \u001b[0m \u001b[0m  0.0002 \u001b[0m \u001b[0m  0.0002 \u001b[0m \u001b[0m   0.0000 \u001b[0m \u001b[0m  0.0001 \u001b[0m \u001b[0m  5.4211 \u001b[0m \u001b[0m  1.1160 \u001b[0m \u001b[0m\n",
+       " \u001b[0m         sortF \u001b[0m \u001b[0m  0.6591 \u001b[0m \u001b[0m  0.1741 \u001b[0m \u001b[0m   0.0110 \u001b[0m \u001b[0m  0.1128 \u001b[0m \u001b[0m  2.5145 \u001b[0m \u001b[0m  1.3458 \u001b[0m \u001b[0m\n",
+       " \u001b[0m          xFwd \u001b[0m \u001b[0m  0.0008 \u001b[0m \u001b[0m  0.0030 \u001b[0m \u001b[0m   0.0002 \u001b[0m \u001b[0m  0.0010 \u001b[0m \u001b[0m  8.8701 \u001b[0m \u001b[0m  1.0646 \u001b[0m \u001b[0m\n"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 3
+    }
+   ],
+   "source": [
+    "using MCMCChains\n",
+    "chn = read(\"chain-file-10_1.jls\", Chains)\n",
+    "summarize(chn)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xFwd = get(chn, :xFwd)[1];\n",
+    "Ig2rev = get(chn, :Ig2rev)[1];"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dif = 0.1\n",
+    "gasPart = 10\n",
+    "gasCur = 0.01\n",
+    "\n",
+    "AXL = 15.5\n",
+    "MerTK = 55\n",
+    "Tyro3 = 3.7\n",
+    "\n",
+    "CompParams = vcat(dif, gasPart, gasCur, xFwd[1], AXL, MerTK, Tyro3, Ig2rev[1], [1.0, 1.0, 1.8, 100.0]);\n",
+    "\n",
+    "tps = @SVector Float64[0.01, 0.1, 1];\n",
+    "r = 1:100;"
+   ]
+  },
+  {
```

Comment:
@ajtsao1 One thing that would be helpful to take care of first is turning all this code into a consistent function. I could see a function where you input the parameters, time points, and the reduction vector, and it produces the plot for you. That would turn much of this notebook into just a few lines.

In the mean time, I'm also digging into the compartmental model a bit more.